sos/plugins/logs.py

```diff
@@ -143,4 +80,32 @@ def setup(self):
                     "/var/log/auth.log*",
                 ])
 
+    def postproc(self):
+        self.do_path_regex_sub(
+            r"/etc/rsyslog*",
+            r"ActionLibdbiPassword (.*)",
+            r"ActionLibdbiPassword [********]"
+        )
+        self.do_path_regex_sub(
+            r"/etc/rsyslog*",
+            r"pwd=.*",
+            r"pwd=[******]"
+        )
+
+
+class RedHatLogs(Logs, RedHatPlugin):
+
+    def setup(self):
+        super(RedHatLogs, self).setup()
+        self.add_journal(boot="this", catalog=True)
+        self.add_journal(boot="last", catalog=True)
+
+        since = ""
+
+        if self.get_option("since"):
+            since = self.get_option("since")
```

Comment:
Could this filter apply not only to the `add_journal` call below, but to *any* journalctl command called? I.e. to https://github.com/sosreport/sos/blob/master/sos/plugins/__init__.py#L1369, add line:

     since = since or self.get_option("since")


Then `--since` option will apply to collecting any journal logs. (an update of `sosreport --help` shall follow, then, currently it states `--since` applies to *files* and not journals; but I think it makes sense to have unified user experience of `--since` option).

Would be that acceptable?

Reply:
Yes..  Resolving this